[2025-02-16T08:22:20.315+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:22:17.131282+00:00 [queued]>
[2025-02-16T08:22:20.332+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:22:17.131282+00:00 [queued]>
[2025-02-16T08:22:20.333+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-02-16T08:22:20.334+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2025-02-16T08:22:20.334+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-02-16T08:22:20.352+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-16 08:22:17.131282+00:00
[2025-02-16T08:22:20.360+0000] {standard_task_runner.py:52} INFO - Started process 481 to run task
[2025-02-16T08:22:20.407+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'pipeline_group1', 'extract_data', 'manual__2025-02-16T08:22:17.131282+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_inka_DAG.py', '--cfg-path', '/tmp/tmp1a68bpvp', '--error-file', '/tmp/tmpfct3busu']
[2025-02-16T08:22:20.420+0000] {standard_task_runner.py:80} INFO - Job 7: Subtask extract_data
[2025-02-16T08:22:20.920+0000] {task_command.py:371} INFO - Running <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:22:17.131282+00:00 [running]> on host 8d7c06d84acb
[2025-02-16T08:22:21.225+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=group1
AIRFLOW_CTX_DAG_ID=pipeline_group1
AIRFLOW_CTX_TASK_ID=extract_data
AIRFLOW_CTX_EXECUTION_DATE=2025-02-16T08:22:17.131282+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-16T08:22:17.131282+00:00
[2025-02-16T08:22:21.241+0000] {base.py:68} INFO - Using connection ID 'postgres_***' for task execution.
[2025-02-16T08:23:26.149+0000] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2025-02-16T08:23:28.629+0000] {taskinstance.py:1412} INFO - Marking task as FAILED. dag_id=pipeline_group1, task_id=extract_data, execution_date=20250216T082217, start_date=20250216T082220, end_date=20250216T082328
[2025-02-16T08:23:30.725+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
